src/App.tsx

```diff
@@ -52,7 +52,9 @@ export const App = () => {
           </Suspense>
           <IntermediateProgress />
           {config.content?.pages.map((page, id) => (
-            <RenderPageRoute key={id} page={page} />
+            <Suspense key={id} fallback={getFallbackComponent()}>
```

Comment:
Not sure if this causes any unwanted side effects? If there is any other solution, feel free to modify or comment. 🙂 

Without `Suspense` component, UI will not render if `useTranslation` hook is in use (error message: `A React component suspended while rendering, but no fallback UI was specified`).